commands/backups/capture.js

```diff
@@ -17,6 +17,21 @@ function * run (context, heroku) {
       yield heroku.post(`/postgres/v0/databases/${db.id}/snapshots`, {host: host(db)})
     }))
   } else {
+    let dbInfo = yield heroku.request({
+      host: host(db),
+      method: 'get',
+      path: `/client/v11/databases/${db.id}`
+    }).catch(err => {
+      if (err.statusCode !== 404) throw err
+      cli.exit(1, `${cli.color.addon(db.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
+    })
+    console.log(dbInfo.info)
+    if (dbInfo) {
+      let dbProtected = /On/.test(dbInfo.info.find(attribute => attribute.name === 'Continuous Protection').values[0])
+      if (dbProtected) {
+        cli.warn('Continuous protection is already enabled for this database. Logical backups of large databases are likely to fail. See https://devcenter.heroku.com/articles/heroku-postgres-data-safety-and-continuous-protection#physical-backups-on-heroku-postgres.')
```

Comment:
let's add new line between `fail.` and `See` (I think just adding `\n` is fine? @uhoh-itsmaciek may know more)

Reply:
Yeah, that'd work. Or you can do two separate `cli.warn`s.